frappe/commands/site.py

```diff
@@ -125,14 +127,18 @@ def restore(context, sql_file_path, mariadb_root_username=None, mariadb_root_pas
 	else:
 		base_path = '.'
 
-
 	if sql_file_path.endswith('sql.gz'):
 		decompressed_file_name = extract_sql_gzip(os.path.abspath(sql_file_path))
 	else:
 		decompressed_file_name = sql_file_path
 
 	site = get_site(context)
 	frappe.init(site=site)
+
+	# dont allow downgrading to older versions of frappe without force
+	if not force and is_downgrade(decompressed_file_name):
+		click.confirm("Downgrading sites may lead to a broken site. Do you wish to continue?", abort=True)
```

Comment:
```suggestion
		click.confirm("Downgrading to an older version may cause unexpected behavior. Do you want to continue?", abort=True)
```

We could also show more information like which app will be downgraded and from which version.

Reply:
this is regarding frappe's compatibility alone. It doesn't check for any other app's

Reply:
@gavindsouza All the more reason to explicitly state that we are checking only the `frappe` version